CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 ## Unreleased
 
 Bugfixes:
+- FrameworkDependencyResolver must not throw NRE if dependency has no type (eg. depending on a named component)
+- Register ViewComponents and TagHelpers correctly
+- Allow crosswiring multiple implementations of the same service
+- Count TagHelper classes with __Generated__ in the name (eg. TagHelpers generated for ViewComponents) as Framework classes
```

Comment:
I think it would be worthwhile prefixing these 4 lines with "ASP.NET Core Facility: " otherwise it makes it hard to read the changelog.

Reply:
Done!